src/main/java/org/folio/rest/support/ResponseUtil.java

```diff
@@ -10,4 +11,12 @@ private ResponseUtil() {}
   public static boolean isUpdateSuccessResponse(Response response) {
     return response != null && response.getStatus() == HTTP_NO_CONTENT.toInt();
   }
+
+  public static boolean isDeleteSuccessResponse(Response response) {
+    return response != null && response.getStatus() == HTTP_NO_CONTENT.toInt();
```

Comment:
The methods seem to share a high degree of commonality, could a method be extracted to check that a response has the expected status code?

Reply:
Shure, extracted.